cmake/execinfo.cmake

```diff
@@ -1,4 +1,6 @@
 #
 # Unwind
 #
-find_package(Libexecinfo QUIET)
+if(NOT MSVC)
```

Comment:
This is strange you need that because this works for PDAL main repo. Is it an MSCV 2017 issue?

Reply:
Yes it is related to MSVC 2017.